packages/client/src/exchanges/specialErrorExchange.ts

```diff
@@ -18,13 +18,18 @@ const specialErrorExchange = ({ intl }: { intl: IntlShape }) => {
         tap((result) => {
           const err = result.error;
           if (err) {
-            const res = err.response as Response;
+            const errRes = err.response as Response;
+            // I think this is old error condition from when the firewall responded directly when blocking
             if (
-              res?.status === 403 &&
+              errRes?.status === 403 &&
               err?.networkError?.message?.includes("Request Rejected")
             ) {
               toast.error(intl.formatMessage(errorMessages.requestRejected));
             }
+            // This is the new error condition from when the firewall responds a redirect to restricted.html
+            if (errRes?.url?.endsWith("/restricted.html")) {
```

Comment:
So, if the current setup **is** working like the Prod firewall does, this still may not work. When I log errRes here, I see
```
redirected: false
status: 405
statusText: "Not Allowed"
type: "basic"
url: "http://localhost:8000/admin/graphql"
```

Reply:
That's weird... it works for me?
![image](https://github.com/user-attachments/assets/8edb0e76-e5cc-4f7b-90da-e265459a7e57)

Reply:
405 "Not Allowed" is usually when you try to GET at the graphql endpoint.  How are you testing this?

Reply:
Rebuilt and tried it again, and it works for me as well now. Maybe I had something set up wrong.